app/assets/javascripts/views/story_view.js

```diff
@@ -569,6 +579,10 @@ module.exports = FormView.extend({
     return [I18n.t('story.state.' + option), option];
   },
 
+  createRequestedByOptions: function(option) {
+    return [option[0], option[1]];
```

Comment:
Do we need this method since `option` is already an array? 🤔 

Maybe at line 553 you could just do:

```javascript
const storyRequestedByOptions = this.model.collection.project.users.forSelect();
```